benchexec/tools/ultimate.py

```diff
@@ -321,18 +317,12 @@ def program_files(self, executable):
         )
         return [executable] + self._program_files_from_executable(executable, paths)
 
-    def determine_result(self, returncode, returnsignal, output, is_timeout):
-        if self._uses_propertyfile:
-            return self._determine_result_with_propertyfile(
-                returncode, returnsignal, output, is_timeout
-            )
-        return self._determine_result_without_propertyfile(
-            returncode, returnsignal, output, is_timeout
-        )
+    def determine_result(self, run):
+        if any([arg for arg in run.commandline if "--spec" == arg or ".prp" in arg]):
```

Comment:
```suggestion
        if any(arg for arg in run.commandline if "--spec" == arg or ".prp" in arg):
```
The tool `flake8` that we use for checking best practices of Python code suggests this, and we use it in CI so we want to apply all of its suggestions when they make sense.

Reply:
Do you have a flake8 config somewhere so that I can integrate it into PyCharm? I ran flake8 3.8.4, and it did not complain about that.

Reply:
Sure, our config is checked in already (`.flake8`). It just does not run on Travis CI due to the inflexibility of Travis' configuration, and GitLab CI unfortunately does not work on GitHub PRs yet. This is why I mention it manually.

If you want to check it for yourself, this is of course nice, and you are probably just missing the package `flake8-awesome` with additional plugins. In our `.gitlab-ci.yml` for job `flake8` it is visible that there is nothing necessary besides installing that package and running `flake8`. (Of course there is no expectation for PR authors to look there, I am just mentioning this as a possible reference.)